recurly/transactions.go

```diff
@@ -17,6 +17,7 @@ type (
 	Transaction struct {
 		XMLName         xml.Name  `xml:"transaction"`
 		Invoice         href      `xml:"invoice,omitempty"`
+		InvoiceNumber   int       `xml:"invoice_number,omitempty"`
```

Comment:
Is this part of just the webhooks? if so, could you remove from here and put just in the webhooks package? API users might expect that value to be populated if the field is on the struct.